webapp/cypress/e2e/batchSampleFeature.cy.js

```diff
@@ -143,7 +143,7 @@ describe("Batch sample creation", () => {
     cy.searchAndSelectItem("component3", "#synthesis-information .vs__search");
     cy.get("#synthesis-information tbody tr:nth-of-type(1) input").first().type("30");
 
-    cy.searchAndSelectItem("component4", "#synthesis-information .vs__search");
+    cy.searchAndSelectItem("component4", "#synthesis-information .vs__search", true);
```

Comment:
What does this do? Should it be everywhere? I can't even find this method in the docs anymore